memory_storage/src/lib.rs

```diff
@@ -246,8 +246,6 @@ pub enum MemoryStorageError {
     UnsupportedMethod,
     #[error("Error serializing value.")]
     SerializationError,
-    #[error("Value does not exist.")]
```

Comment:
This is a public enum, so a breaking change as well that we should add in the changelog description.